packages/zoe/src/contracts/sellItems.js

```diff
@@ -0,0 +1,92 @@
+/* eslint-disable no-use-before-define */
+// @ts-check
+
+import harden from '@agoric/harden';
+import { makeZoeHelpers, defaultAcceptanceMsg } from '../contractSupport';
+
+/** @typedef {import('../zoe').ContractFacet} ContractFacet */
+
+// zcf is the Zoe Contract Facet, i.e. the contract-facing API of Zoe
+export const makeContract = harden(
+  /** @param {ContractFacet} zcf */ zcf => {
+    const { assertKeywords, rejectOffer, checkHook } = makeZoeHelpers(zcf);
+    assertKeywords(harden(['Items', 'Money']));
+
+    const { terms } = zcf.getInstanceRecord();
+    const { pricePerItem } = terms;
+
+    let sellerOfferHandle;
+
+    const sellerOfferHook = offerHandle => {
+      sellerOfferHandle = offerHandle;
+      return defaultAcceptanceMsg;
+    };
+
+    const buyerOfferHook = buyerOfferHandle => {
+      const sellerAllocation = zcf.getCurrentAllocation(sellerOfferHandle);
```

Comment:
I was originally thinking that I wanted to see this fetch when the seller is setup, but of course that's ridivulous, since this needs to get the allocation *now*.  Are we in position to break an allocation description after it's obsolete, so that instead of making a bad trade against data from a prior turn, we actaully require that code retrieve it from handles from scratch?

Reply:
I don't see a good mechanism to ensure that the contract has the latest allocation data, but having delta helpers might be the way to do that.